qa/rpc-tests/bip68-112-113-p2p.py

```diff
@@ -0,0 +1,547 @@
+        success_txs.extend(all_rlt_txs(bip112txs_vary_nSequence_v1))
+        success_txs.extend(all_rlt_txs(bip112txs_vary_OP_CSV_v1))
+        # try BIP 112 with seq=9 txs
+        success_txs.extend(all_rlt_txs(bip112txs_vary_nSequence_9_v1))
+        success_txs.extend(all_rlt_txs(bip112txs_vary_OP_CSV_9_v1))
+        yield TestInstance([[self.create_test_block(success_txs), True]]) # 6
+        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+
+        ### Version 2 txs ###
+        success_txs = []
+        # add BIP113 tx and -1 CSV tx
+        bip113tx_v2.nLockTime = self.last_block_time - 600 * 5 # = MTP of prior block (not <) but < time put on current block
+        bip113signed2 = self.sign_transaction(self.nodes[0], bip113tx_v2)
+        success_txs.append(bip113signed2)
+        success_txs.append(bip112tx_special_v2)
+        # add BIP 68 txs
+        success_txs.extend(all_rlt_txs(bip68txs_v2))
+        # add BIP 112 with seq=10 txs
+        success_txs.extend(all_rlt_txs(bip112txs_vary_nSequence_v2))
+        success_txs.extend(all_rlt_txs(bip112txs_vary_OP_CSV_v2))
+        # try BIP 112 with seq=9 txs
+        success_txs.extend(all_rlt_txs(bip112txs_vary_nSequence_9_v2))
+        success_txs.extend(all_rlt_txs(bip112txs_vary_OP_CSV_9_v2))
+        yield TestInstance([[self.create_test_block(success_txs), True]]) # 7
+        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+
+
+        # 1 more version 4 block to get us to height 575 so the fork should now be active for the next block
+        test_blocks = self.generate_blocks(1, 4)
+        yield TestInstance(test_blocks, sync_every_block=False) # 8
+        assert_equal(self.get_bip9_status('csv')['status'], 'active')
+
+
+        #################################
+        ### After Soft Forks Activate ###
+        #################################
+        ### BIP 113 ###
+        # BIP 113 tests should now fail regardless of version number if nLockTime isn't satisfied by new rules
+        bip113tx_v1.nLockTime = self.last_block_time - 600 * 5 # = MTP of prior block (not <) but < time put on current block
+        bip113signed1 = self.sign_transaction(self.nodes[0], bip113tx_v1)
+        bip113tx_v2.nLockTime = self.last_block_time - 600 * 5 # = MTP of prior block (not <) but < time put on current block
+        bip113signed2 = self.sign_transaction(self.nodes[0], bip113tx_v2)
+        for bip113tx in [bip113signed1, bip113signed2]:
+            yield TestInstance([[self.create_test_block([bip113tx]), False]]) # 9,10
+        # BIP 113 tests should now pass if the locktime is < MTP
+        bip113tx_v1.nLockTime = self.last_block_time - 600 * 5 - 1 # = MTP of prior block (not <) but < time put on current block
+        bip113signed1 = self.sign_transaction(self.nodes[0], bip113tx_v1)
+        bip113tx_v2.nLockTime = self.last_block_time - 600 * 5 - 1 # = MTP of prior block (not <) but < time put on current block
```

Comment:
nit: The comments at lines 386 and 388 appear to be incorrect.  Perhaps `# < MTP of prior block`?